index.rst

```diff
@@ -143,104 +164,207 @@ Core Components
 Sensor Components
 -----------------
 
+Sensors have been split into categories. If a sensor fits into more than one category, it will be listed multiple times.
+
+Core / Internal
+***************
 .. imgtable::
 
     Sensor Core, components/sensor/index, folder-open.svg
-    ADC, components/sensor/adc, flash.svg
-    ADE7953, components/sensor/ade7953, ade7953.svg
-    ADS1115, components/sensor/ads1115, ads1115.jpg
-    AHT10, components/sensor/aht10, aht10.jpg
-    AM2320, components/sensor/am2320, am2320.jpg
-    APDS9960, components/sensor/apds9960, apds9960.jpg
-    ATM90E32, components/sensor/atm90e32, atm90e32.jpg
-    AS3935, components/sensor/as3935, as3935.jpg
-    BH1750, components/sensor/bh1750, bh1750.jpg
-    Binary Sensor Map, components/sensor/binary_sensor_map, binary_sensor_map.jpg
-    BLE RSSI, components/sensor/ble_rssi, bluetooth.svg
-    BLE Client Sensor, components/sensor/ble_client, bluetooth.svg
-    BME280, components/sensor/bme280, bme280.jpg
-    BME680, components/sensor/bme680, bme680.jpg
-    BME680 via BSEC, components/sensor/bme680_bsec, bme680.jpg
```

Comment:
This is a separate component and has gone missing in the shuffle.

Reply:
oops, added back in! thank you